packages/app/components/Today/ChatbotToday.tsx

```diff
@@ -186,9 +192,22 @@ export const ChatbotToday: React.FC = () => {
                       <Avatar size="small" icon={<RobotOutlined />} />
                       <div className="ml-2 flex flex-col gap-1">
                         <div className="flex items-start gap-2">
-                          <div className="max-w-[280px] rounded-xl bg-slate-100 px-3 py-2">
-                            {' '}
+                          <div
+                            className={`max-w-[280px] rounded-xl px-3 py-2 ${
+                              item.verified ? 'bg-green-100' : 'bg-slate-100'
+                            }`}
+                          >
                             {item.message}
+                            {item.verified && (
+                              <div className=" rounded-xl bg-slate-100">
+                                <Tooltip title="A similar question has been asked before, and the answer has been verified by a faculty">
+                                  <CheckCircleOutlined
+                                    style={{ color: 'green', fontSize: '20px' }}
+                                  />
+                                  <span> See why this is verified </span>
+                                </Tooltip>
+                              </div>
```

Comment:
```suggestion
                              <Tooltip title="A similar question has been asked before, and the answer has been verified by a faculty member">
                                  <CheckCircleOutlined
                                    style={{ color: 'green', fontSize: '20px', marginLeft: '2px'}}
                                  />
                                </Tooltip>
```

I think I like this a little more. Personally, I don't think the "See why this is verified" text is very necessary, and I think it could probably be removed to reduce visual clutter.